test/integration/test_proxies.py

```diff
@@ -158,6 +160,21 @@ def test_proxy_urls(self, qe_token, qe_url):
                 )
                 version_finder.version()
 
+    @skipIf(requests_version > "2.26", "Need requests <= 2.26")
+    @requires_qe_access
+    def test_proxy_urls_2(self, qe_token, qe_url):
+        """Test proxy urls only supported by requests <= 2.26"""
+        test_urls = ["http:{}:{}".format(ADDRESS, PORT)]
+        for proxy_url in test_urls:
+            with self.subTest(proxy_url=proxy_url):
+                credentials = Credentials(
+                    qe_token, qe_url, proxies={"urls": {"https": proxy_url}}
+                )
+                version_finder = VersionClient(
+                    credentials.base_url, **credentials.connection_parameters()
+                )
+                version_finder.version()
+
```

Comment:
I don't see much value in porting this PR as-is because this will add a test case that will always be skipped (`requests` currently resolves to `2.27.1` on a fresh install).

@kt474 - Can you check if there is another way to make the test compatible with most recent request versions?

@rathishcholarajan - please jump in if you think this is not a reasonable proposal. I would have probably given similar feedback for https://github.com/Qiskit/qiskit-ibmq-provider/pull/1103.

Reply:
I don't believe there is another way to test the `"http:{}:{}"` format, but yes this test case will always be skipped. We still need these changes so `test_proxy_urls` doesn't fail - but I don't think this test case is necessary anyway since the url format is not supported 

@rathishcholarajan thoughts?

Reply:
Sure we can remove this test